switchery.js

```diff
@@ -75,7 +75,12 @@ function Switchery(element, options) {
  */
 
 Switchery.prototype.hide = function() {
-  this.element.style.display = 'none';
+  this.element.style.height = '100%';
+  this.element.style.width = '100%';
+  this.element.style.opacity = '.0001';
+  this.element.style.position = 'absolute';
+  this.element.style.left = '0';
+  this.element.style.top = '0';
```

Comment:
📓 (note)

A convenient way to set a bunch of styles on a node with CSS-like syntax is to make use of `Object.assign`:

```js
Object.assign(this.element.style, {
  bottom: '100%',
  overflow: 'hidden',
  position: 'fixed',
  right: '100%'
});
```